src/cmdlib.sh

```diff
@@ -582,24 +582,9 @@ runcompose_extensions() {
 }
 
 runvm_with_cache() {
-    # "cache2" has an explicit label so we can find it in qemu easily
-    if [ ! -f "${workdir}"/cache/cache2.qcow2 ]; then
-        qemu-img create -f qcow2 cache2.qcow2.tmp 10G
-        (
-         # shellcheck source=src/libguestfish.sh
-         source /usr/lib/coreos-assembler/libguestfish.sh
-         virt-format --filesystem=xfs --label=cosa-cache -a cache2.qcow2.tmp)
-        mv -T cache2.qcow2.tmp "${workdir}"/cache/cache2.qcow2
-    fi
-    # And remove the old one
+    # Clean up old cache qcow2; we now always rely on virtiofs
```

Comment:
Since CI failed and we'll need to do another run.. maybe we can add a comment here to say to delete the `runvm_with_cache` function after 2023 to let future us know for sure when it's safe to get rid of it.